store/storetest/post_store.go

```diff
@@ -1995,6 +2002,15 @@ func testPostStoreGetFlaggedPostsForTeam(t *testing.T, ss store.Store, s SqlStor
 	require.NoError(t, err)
 	time.Sleep(2 * time.Millisecond)
 
+	// Post on channel where user is not a member
+	o6 := &model.Post{}
+	o6.ChannelId = model.NewId()
+	o6.UserId = m2.UserId
+	o6.Message = NewTestId()
+	o6, err = ss.Post().Save(o6)
+	require.NoError(t, err)
+	time.Sleep(2 * time.Millisecond)
```

Comment:
I realize these lines were already there from before. But do you know of the reason? It's not good to be indefinitely adding sleeps.

Reply:
My wild guess is to deal with read replica situations, but it is just a guess.

Reply:
There are no read replicas in tests. I'd be willing to remove the sleep and then see what happens, and fix that. But anyways, outside the scope of this PR.